README.md

```diff
@@ -242,6 +238,13 @@ either you or they can follow these steps:
 
 ## Enabling optional features
 
+### Using GPT-4
```

Comment:
This was an FAQ, moved it up here. FAQs are hard to link to, the way we implemented them, sections with proper headings are more usable.